spec/unit/content_item_publisher/finder_email_signup_publisher_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "spec_helper"
+
+RSpec.describe ContentItemPublisher::FinderEmailSignupPublisher do
+  subject(:instance) { described_class.new(finder, timestamp) }
+
+  let(:config_file) { "finders/news_and_communications_email_signup.yml" }
+  let(:finder) { YAML.load_file(File.join(Dir.pwd, "config", config_file)) }
+  let(:content_id) { finder["content_id"] }
+  let(:timestamp) { Time.now.iso8601 }
+  let(:logger) { instance_double("Logger") }
+
+  before do
+    allow(Logger).to receive(:new).and_return(logger)
+  end
+
+  describe "#call" do
+    let(:publishing_api) { instance_double("GdsApi::PublishingApiV2") }
+    let(:payload) {
+      ContentItemPublisher::FinderEmailSignupPresenter.new(finder, timestamp).present
+    }
+
+    before do
+      allow(logger).to receive(:info)
+      allow(GdsApi::PublishingApiV2).to receive(:new).and_return(publishing_api)
```

Comment:
- Line 24 could be a bit different when using Services.publishing_api
- Could think about using test helpers defined in: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/test_helpers/publishing_api_v2.rb